.github/workflows/docker.build.yml

```diff
@@ -54,4 +54,75 @@ jobs:
           cache-to: type=local,mode=max,dest=/tmp/.buildx-cache
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
-          labels: ${{ steps.docker_meta.outputs.labels }}
\ No newline at end of file
+          labels: ${{ steps.docker_meta.outputs.labels }}
+
+      -
+        name: Check whether wbaas-deploy should be updated
```

Comment:
An alternative to doing this check and then skipping the following steps would be to stop releasing `latest` when pushing to `main` which I think we don't ever use anywhere anyways?

Reply:
yeah, we don't use it anyway either AFAICT; I believe we should therefore feel free to remove it.

Reply:
ah, yeah. I believe we definitely want to check this builds on each PR (otherwise there is a high chance of us merging non-building code).

One alternative if you wanted to refactor more heavily would be to:
- stick these following steps in their own job.
- Make this job dependent on the build job running (https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow#example-requiring-successful-dependent-jobs)
-  using a condition on the job to ensure it only runs on tags that exist on `main` (https://docs.github.com/en/actions/using-jobs/using-conditions-to-control-job-execution)